### ISSUE TITLE
ImplementInterface Refactoring does not apply all Vertical Spacing settings

### ISSUE THREAD

**Issue by BZngr** [rubberduck-vba/Rubberduck#5673]
Version 2.5.1.20527
OS: Microsoft Windows NT 10.0.18363.0, x64
Host Product: Microsoft Office x64
Host Version: 16.0.13628.20380
Host Executable: EXCEL.EXE

**Description**
Property blocks generated by the `ImplementInterface` refactoring does not apply the 'Remove vertical space between related property members' flag.

**To Reproduce**
1. Go to `Indenter` 'Vertical Spacing' make sure the 'Remove vertical space between related property members' checkbox is checked.
2. Use RD refactoring to implement an interface that defines Let/Set and Get Property procedures.
3. Blank lines will be inserted between related property procedures.

**Expected behavior**
Property procedures of the same `Identifier` should be generated without a blank line between procedures.

**Comment by SmileyFtW** [comment]
What if no blank lines between all property procedures is the desired outcome? This seems to be only for related `Get Let Set`.

**Comment by BZngr** [comment]
Yes - the recent `Indenter` setting change regarding Vertical Spacing only applied to  _adjacent and related_ groups of Properties.  So, providing an option to remove blank lines between all adjacent Property-type procedures regardless of `Identifier` would require an additional enhancement.
